client/UI/FFRKViewInventory.cs

```diff
@@ -116,6 +118,34 @@ private void FFRKViewInventory_Load(object sender, EventArgs e)
             }
         }
 
+        private void FFRKViewInventory_Recalc()
+        {
+            if (DesignMode)
+                return;
+
+            if (FFRKProxy.Instance != null)
+            {

```

Comment:
Are you sure you need these lines?  The only place this should matter is in the FFRKViewInventory_Load() function, because it can be called by the designer (i.e. when you're in Visual Studio and you load the UI editor).  So it exists in the Load function so that when the designer calls it it doesn't crash, but as long as _Load() doesn't call _Recalc in an unprotected block, this should be unnecessary.

Also just as a matter of style, I only really use the _Foo() naming convention for event handlers.  For regular helper functions and what not just call it something like Recalculate()